packages/next/src/server/lib/incremental-cache/fetch-cache.ts

```diff
@@ -23,6 +31,18 @@ const CACHE_REVALIDATE_HEADER = 'x-vercel-revalidate' as const
 const CACHE_FETCH_URL_HEADER = 'x-vercel-cache-item-name' as const
 const CACHE_CONTROL_VALUE_HEADER = 'x-vercel-cache-control' as const
 
+const zCachedFetchValue: zod.ZodType<CachedFetchValue> = z.object({
```

Comment:
If you're casting it lower, use the direct type here:

```suggestion
const zCachedFetchValue: zod.ZodType<IncrementalCacheValue> = z.object({
```

Reply:
Aren't we using the direct type → https://github.com/vercel/next.js/blob/27c9fb755a06fdc2aeccbf3b4e5e18e0103e1d5c/packages/next/src/server/response-cache/types.ts#L96?